carta/image.py

```diff
@@ -477,17 +477,21 @@ def set_scaling(self, scaling, alpha=None, gamma=None, min=None, max=None):
             The alpha value (only applicable to ``LOG`` and ``POWER`` scaling types).
         gamma : {2}
             The gamma value (only applicable to the ``GAMMA`` scaling type).
-        min : {3}
-            The minimum of the scale. Only used if both *min* and *max* are set.
-        max : {4}
-            The maximum of the scale. Only used if both *min* and *max* are set.
+        rank : {3}
+            The percentile rank. If this is set, *min* and *max* are ignored, and will be calculated automatically.
```

Comment:
Renaming this for consistency is a good idea, but I think the complete absence of the word "clip" makes the naming of this set of parameters a little confusing. I would add it in this line only: `The clip percentile rank...`.

I think that for clarity and consistency we should also edit the description of `set_percentile_rank` so that the first line is `Set the clip percentile.` The function should perhaps be called `set_clip_percentile` (the parameter would still be called `rank`). This would change the API, and would affect any existing code. @kswang1029 , what do you think?

Reply:
In 48db542, the `set_percentile_rank` is renamed with `set_clip_percentile`. The descriptions are modified according to @confluence's suggestions.